tests/phpunit/IncrementalFetchTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractor\Tests;
+
+use Keboola\DbExtractor\Exception\UserException;
+use Keboola\DbExtractor\Tests\Traits\CreateApplicationTrait;
+use PHPUnit\Framework\TestCase;
+
+class IncrementalFetchTest extends TestCase
+{
+    use CreateApplicationTrait;
+
+    /**
+     * @dataProvider invalidConfigProvider
+     */
+    public function testUserError(string $expectedMsg, array $parameters): void
+    {
+        $config = $this->getConfig();
+        $config['parameters'] = $parameters + $config['parameters'];
+        $this->expectException(UserException::class);
+        $this->expectExceptionMessage($expectedMsg);
+        $this->createApplication($config)->run();
+    }
+
+    public function invalidConfigProvider(): array
+    {
+        return [
+            'missing column' => [
+                'Incremental fetching column [foo] not found.',
```

Comment:
Myslím, že jsme se někde dohodli, že budeme používat sprintf('Table "%s" cannnot be found', 'xy'). Místo hranatých závorek.

Reply:
Ok. Toto som prebral z ineho repa, `db-extractor-impala`, dam tam vsade uvodzovky.

Reply:
jj, jasně - v těch db-ex je binec v tomhle... a je to spíš "convention" a chápu, že nemáš šanci vymyslet, který z těch milionu způsobů, co se dá najít je ten "správný" :D